models/models/web.py

```diff
@@ -117,8 +117,3 @@ class ProjectSummary(SMBase):
     seqr_sync_types: list[str]
 
     links: PagingLinks | None
-
-    class Config:
-        """Config for ProjectSummaryResponse"""
-
-        fields = {'links': '_links'}
```

Comment:
This change I am the least sure about. I think this is left over from draft versions of this code — `_links` doesn't appear anywhere else (in metamist) any more.

Reply:
Yeah fun, happy for this change :) we were going to go links, but added page tokens (which is effectively just the sample ID)

Reply:
The `links` field should still be there and the code initialising it is still there; this removal just removes the aliasing of it so that it's really called `_links` — which wasn't working because nothing else thinks it's called `_links`. Or something like that — I didn't find the pydantic field aliasing documentation very clear.

But anyway, sounds like this field doesn't matter too much => even better!